Zend/zend_inheritance.c

```diff
@@ -477,8 +477,10 @@ static inheritance_status zend_is_class_subtype_of_type(
 	zend_class_entry *fe_ce = NULL;
 	bool have_unresolved = 0;
 
+	uint32_t proto_type_full_mask = ZEND_TYPE_FULL_MASK(proto_type);
+
```

Comment:
IMO this change is not useful, as we're barely saving characters but introducing more reader indirection.